.github/pull_request_template.md

```diff
@@ -1,23 +1,21 @@
 Fixes
 -----
-
-Fixes WB-NNNN
-
-Fixes #NNNN
+- Fixes WB-NNNNN
```

Comment:
Were these changes an intentional part of this PR?

Reply:
yes, the laziness to submit another pr was quite intentional ^_^